radicle-cli/Cargo.toml

```diff
@@ -15,6 +15,8 @@ log = { version = "0.4", features = ["std"] }
 serde_json = { version = "1" }
 thiserror = { version = "1" }
 zeroize = { version = "1.1" }
+json-color = "0.7"
+chrono = { version = "0.4", default-features = false, features = ["clock", "std"] }
```

Comment:
nit: these deps were in alphabetical order :)

Reply:
Fixed. Do you use any tools for this? Found [cargo-sort](https://crates.io/crates/cargo-sort).

Reply:
Other than my eyes, nope :smile: `cargo-sort` looks good though :)